tools/cli/commands/release.js

```diff
@@ -150,12 +155,39 @@ export async function scriptRouter( argv ) {
 				  - You will now likely want to start a new release cycle like so:
 				      jetpack release ${ argv.project } new-cycle \n`.replace( /^\t+/gm, '' );
 			break;
-		case 'append':
-		case 'new-cycle':
+		case 'amend':
+			await checkBranchValid( argv );
+			argv.script = `vendor/bin/changelogger`;
+			argv.scriptArgs = [ `write`, `--amend` ];
+			argv.workingDir = `projects/${ argv.project }`;
+			argv.next = `Finished! Next:  
+				  	- You will now likely want to update readme.txt again, then commit to the release branch:
+				    	jetpack release ${ argv.project } readme \n`.replace( /^\t+/gm, '' );
+			break;
+		case 'version':
 			console.log( `${ argv.script } is not implemented yet!` );
 			process.exit( 1 );
 	}
 }
+
+/**
+ * Checks and makes sure we're on a release branch before we run the `amend` script.
+ *
+ * @param {object} argv - the arguments passed.
+ */
+export async function checkBranchValid( argv ) {
+	const currentBranch = child_process.execSync( 'git branch --show-current' ).toString().trim();
+	const proj = argv.project.split( '/' )[ 1 ]; // get project without project type attached.
```

Comment:
Won't work for `plugins/beta`. What you need to to be completely correct is to check the project's composer.json for `.extra.release-branch-prefix`. If that's not set, then always fail.

Reply:
Ah, thanks! Should be fixed.